### PR TITLE
    Fix for DF-21

### PR DESCRIPTION
**Pull request by jitupawar22**
Added mark.auth tag for various auth related tests.
    Fixed various errors from different files.
    Removed unnecessary asserts from test.utils
    Commented out logging statements which generates huge logs

Signed-off-by: Jitendra Pawar <jitendra@portworx.com>

### CODE REVIEW

File: testing/sdk_tasks.py [hunk 1]
```diff
@@ -248,11 +248,12 @@ def get_summary(with_completed: bool = False, task_name: Optional[str] = None) -
     )
     if task_name:
         output = list(filter(lambda t: t.name == task_name, output))
-    log.info(
-        "Task summary (with_completed={}, task_name=[{}]):\n- {}".format(
-            with_completed, task_name, "\n- ".join([str(e) for e in output])
-        )
-    )
+    # Commenting below two lines to avoid huge output
```
Comment (disrani-px) [inline testing/sdk_tasks.py]: Change the log to debug instead of commenting it out

**Review by disrani-px — COMMENTED** [review]
Looks like there are a few merge conflicts

File: testing/sdk_tasks.py (hunk 1 above)
Comment (jitupawar22) [reply testing/sdk_tasks.py]: This is useless huge log. We earlier also commented out in the dcos-commons repo. Finding real problem does not help with this log. It just floods the output and increase log file size.

Comment (disrani-px) [reply testing/sdk_tasks.py]: Debug logs aren't printed by default, right? So you can change it to debug, it won't be printed then

Comment (jitupawar22) [reply testing/sdk_tasks.py]: I did not see any log.debug method, please point me to the method you wanted to use here.

Comment (disrani-px) [reply testing/sdk_tasks.py]: It is part of the logger. You can just use `log.debug()`

https://docs.python.org/2/library/logging.html#logging.debug

**Review by disrani-px — APPROVED** [review]

**Comment by disrani-px** [comment]
@jitupawar22 Can you rebase your changes? There are merge conflicts

**Pull request merged**